documentation/policies.rst

```diff
@@ -82,3 +82,56 @@ Deploy the application and enjoy !
 
 .. |TopologyEditorPolicies| image:: _static/img/topology-policies-button.png
                   :alt: policies button
+
+Applying HTTP Monitoring policy on a web application
+----------------------------------------------------
+
+This chapter presents how to apply an HTTP Monitoring policy on a web application in order to be informed when the web server
+is down.
+
+This can be done using ``yorc.policies.monitoring.HTTPMonitoring`` policy on any location.
+
+A valid target type for applying ``yorc.policies.monitoring.HTTPMonitoring`` is ``tosca.nodes.SoftwareComponent`` .
+
+Only one monitoring policy can be applied on a node template instance.
+
+Let's deploy the ``Welcome Application`` as seen before :ref:`here <welcome_app_section>` and let's configure our HTTP monitoring policy.
+
+Select your application and go to the ``Topology Editor``.
+
+Click on the |TopologyEditorPolicies| button on the vertical blue bar on the left.
+
+Click on the ``+ Add policies`` button, search the policy node ``httpMonitoring`` (yorc.policies.monitoring.HTTPMonitoring) from yorc-types and drag-and-drop it on the policies list of your topology.
+
+Next, you just need to add the ``Welcome`` node template as target of the policy and set the policy properties as below:
+
+The ``scheme`` property allows to select http or https protocol.
+
+The ``port`` property allows to set the port of your web application you want to monitor.
+
+The ``path`` and ``http_headers`` properties allows to specify an URL path and HTTP headers to use for sending request to the web application.
+
+The ``time_interval`` property let you define how often the application must be checked.
+
+.. image:: _static/img/configure-http-monitoring-policy.png
+   :alt: Configure HTTP Monitoring policy
+   :align: center
+
+Once the application is deployed, you can stop the ``Welcome`` web server by running the ``StopWebServer`` custom workflow and depending the time_interval you specified,
+you can rapidly (or not...) observe the Welcome node in Stopped State in the runtime view and the following log:
+
+      ``[2019-04-09 15:15:31] [] [Welcome] Monitoring Check returned a failure for node (Welcome-0)``
+
+.. image:: _static/img/welcome-stopped.png
+   :alt: Welcome component in stopped state
+   :align: center
+
+Next, by running the ``startWebServer`` custom workflow, you restart the web server and can observe the ``Welcome`` node is backed to normal:
+
+     ``[2019-04-10 11:12:24] [] [Welcome] Monitoring Check is back to normal for node (Welcome-0) ``
+
+.. image:: _static/img/welcome-ok.png
+   :alt: Welcome component in started state
+   :align: center
+
+The next step will be to fix this by a self-healing policy !
```

Comment:
As in previous releases, the user who wanted to monitor Compute Instances liveness had to set a  Monitoring Interval parameter in the UI outside of the Topology just before deploying the application, and this parameter will disappear in this release, it may be good to have a specific section here to describe this has changed in this release and describe the way to monitor compute instances now is done by adding a TCPMonitor policy to Compute Instances within the topology.

Reply:
I have added the section "Applying TCP Monitoring policy" to the documentation chapter called "Applying policies"